package.json

```diff
@@ -32,13 +32,10 @@
     "eslint-plugin-cypress": "^2.11.2",
     "eslint-plugin-es": "^4.1.0",
     "eslint-plugin-jest": "^24.3.2",
-    "eslint-plugin-jsdoc": "^32.3.0",
+    "eslint-plugin-jsdoc": "^40.1.1",
     "eslint-plugin-svelte3": "^3.1.1"
   },
   "devDependencies": {
-    "eslint": "^7.22.0"
-  },
-  "peerDependencies": {
-    "eslint": "^7.22.0"
```

Comment:
**issue:** We need either to keep the peer dependency as this package was meant for configuration and we still need to declare which version is recommended for the consumers. Or we must either move the eslint to transitive dependencies.

Reply:
indeed. checked peerDependencies for all dependencies involved:

- @babel/eslint-parser: eslint: ^7.5.0 || ^8.0.0
- eslint-plugin-cypress: eslint: >= 3.2.1
- eslint-plugin-es: eslint: >=4.19.1
- eslint-plugin-jest: eslint: >=5
- eslint-plugin-jsdoc: eslint: ^7.0.0 || ^8.0.0
- eslint-plugin-svelte3: eslint: >=6.0.0

so peerDependencies of eslint ^8.0.0 satisfies all of them - updated the code accordingly (before, with the lower eslint-plugin-jsdoc version it was not the case so eslint ^7 was used).